include/cron.php

```diff
@@ -106,8 +107,8 @@ function cron_run(&$argv, &$argc){
 	// Check every conversation
 	ostatus::check_conversations(false);
 
-	// Set the gcontact-id in the item table if missing
-	item_set_gcontact();
+	// Do post update functions
```

Comment:
little better description would be nice. `Do post update functions` explains nothing